Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'cocoapods'
 gem 'cocoapods-update-if-you-dare', '~> 0.1.1'
+gem 'colored'
```

Comment:
@garriguv I think there is no need for another PR. Just remove colored here from this one and update bundle once again and we're good to go 🚀